### PR TITLE
Correct stage task parameter order and update stage and deploy configurations

### PR DESCRIPTION
**Pull request by loosebazooka**
change config param applicationID to application
change -A to --application

### CODE REVIEW

**Comment by loosebazooka** [comment]
@meltsufin

**Comment by loosebazooka** [comment]
It actually looks like application doesn't even do anything here? Should we just not expose it?

**Comment by meltsufin** [comment]
@loosebazooka How are you determining the command syntax and whether parameters are used?

**Comment by loosebazooka** [comment]
Stage syntax is just appcfg [options] stage source destination

**Comment by loosebazooka** [comment]
@meltsufin ptal

File: src/main/java/com/google/cloud/tools/app/impl/config/DefaultStageStandardConfiguration.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
   private boolean enableQuickstart;
   private boolean disableUpdateCheck;
   private String version;
-  private String applicationId;
+  private String application;
```
Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/app/impl/config/DefaultStageStandardConfiguration.java]: Did you mean to remove application altogether?

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/config/DefaultStageStandardConfiguration.java]: oops yes

Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/app/impl/config/DefaultStageStandardConfiguration.java]: updated

**Comment by meltsufin** [comment]
:ship:

**Pull request merged**